recipe/meta.yaml

```diff
@@ -11,46 +11,64 @@ source:
   sha256: 2ad7f0c8010ebbc46319cc30febfb8e59ccf84969a201541c207bc3a4fa6cf81
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
```

Comment:
Actually this seems to be the reason for the failing tests as well... @ocefpaf Do you know why this is happening since the docs mention that we should use `{{ PYTHON }}`?

![image](https://github.com/conda-forge/altair-feedstock/assets/4560057/ca8aa536-b558-4609-bcdf-020f48692e8a)

Reply:
Thank you @ocefpaf , I just tried re-rendering and I'm still seeing the same error in the pipelines https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=895875&view=results